packages/grafana-data/src/themes/createV1Theme.ts

```diff
@@ -250,3 +217,36 @@ const commonColorsPalette = {
   // New reds palette used by next-gen form elements
   red88: '#e02f44',
 };
+
+export const basicColors = {
+  ...commonColorsPalette,
+  black: '#000000',
+  white: '#ffffff',
+  dark1: '#141414',
+  dark2: '#161719',
```

Comment:
hmmm, unsure about essentially removing these from the theme. 

I think they should have a home in Theme2 - probably identify where they're used and give them semantic names (in theme.components perhaps), or update where they're used to grab them from a more suitable place from Theme2.

Maybe for the time being the change to this file could/should be reverted, and consumers grab them from `theme2.v1.basicColors` or whatever, and tackle these separately - i think that would still be a marginal improvement.